### PR TITLE
Welcome views done: Welcome view, Auth View and Profile picker view

### LINKED ISSUE #8: Login View Swift App

### LINKED ISSUE #10: Profile picker Swft App

### PR DESCRIPTION
**Pull request by Daniel7578**

### CODE REVIEW

File: SwiftApp/ViewModels/AuthenticationManager.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+//  AuthenticationManager.swift
+//  SwiftApp
+//
+//  Created by Daniel Arango Cruz on 18/09/24.
+//
+
+import Foundation
+
+final class AuthenticationManager {
+    static let shared = AuthenticationManager()
```
Comment (storres21) [inline SwiftApp/ViewModels/AuthenticationManager.swift]: This file is empy, should we keep it ?

**Review by storres21 — CHANGES_REQUESTED** [review]
Check the comments

File: SwiftApp/ViewModels/AuthenticationManager.swift (hunk 1 above)
Comment (Daniel7578) [reply SwiftApp/ViewModels/AuthenticationManager.swift]: That's true I forgot to delete it

**Review by storres21 — APPROVED** [review]
Well done!!

**Pull request merged**